<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xfQuestionDlg</class>
 <widget class="QDialog" name="xfQuestionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: qlineargradient(spread:pad, x1:0.191, y1:0.181818, x2:1, y2:0.722, stop:0 rgba(252, 252, 252, 255), stop:1 rgba(165, 165, 165, 255));
	border-color: rgb(0, 0, 0);
	margin: 5px 5px;
	border-style: inset;
	border-radius: 8px;
	border-width: 3px;
}</string>
  </property>
  <widget class="QLabel" name="pMSGLab">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>15</y>
     <width>280</width>
     <height>85</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>line1
line2
line3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>117</y>
     <width>282</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_21">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="xf_resources.qrc">:/images/emergency_button.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="pCancelTB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>196</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QToolButton{
     background-color: qlineargradient(spread:pad, x1:0.191, y1:0.181818, x2:1, y2:0.722, stop:0 rgba(252, 252, 252, 255), stop:1 rgba(165, 165, 165, 255));
     border-width: 3px;
     border-style: outset;
     border-radius: 15px;
     border-color: #777;
	 color: #000;
     font: bold 16px;
     min-width: 10em;
     padding: 0px;
 }
QToolButton:!enabled{
background-color: qlineargradient(spread:pad, x1:0.246175, y1:0.255, x2:1, y2:0.966, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(255, 255, 255, 255));
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #777;
    min-width: 10em;
     padding: 0px;
 } 

 QToolButton:pressed {
     background-color: #555;
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #fff;
    min-width: 10em;
     padding: 0px;
 } 
 QToolButton:checked {
     background-color: #555;
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #fff;
    min-width: 10em;
     padding: 0px;
 } 
</string>
      </property>
      <property name="text">
       <string>No</string>
      </property>
      <property name="iconSize">
       <size>
        <width>280</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>197</y>
     <width>282</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="xf_resources.qrc">:/images/play_button.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="pAcceptTB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>196</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QToolButton{
     background-color: qlineargradient(spread:pad, x1:0.191, y1:0.181818, x2:1, y2:0.722, stop:0 rgba(252, 252, 252, 255), stop:1 rgba(165, 165, 165, 255));
     border-width: 3px;
     border-style: outset;
     border-radius: 15px;
     border-color: #777;
	 color: #000;
     font: bold 16px;
     min-width: 10em;
     padding: 0px;
 }
QToolButton:!enabled{
background-color: qlineargradient(spread:pad, x1:0.246175, y1:0.255, x2:1, y2:0.966, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(255, 255, 255, 255));
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #777;
    min-width: 10em;
     padding: 0px;
 } 

 QToolButton:pressed {
     background-color: #555;
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #fff;
    min-width: 10em;
     padding: 0px;
 } 
 QToolButton:checked {
     background-color: #555;
     border-width: 3px;
     border-style: inset;
     border-radius: 15px;
     border-color: #777;
     font: bold 16px;
 	 color: #fff;
    min-width: 10em;
     padding: 0px;
 } 
</string>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
      <property name="iconSize">
       <size>
        <width>280</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="xf_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pCancelTB</sender>
   <signal>clicked()</signal>
   <receiver>xfQuestionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pAcceptTB</sender>
   <signal>clicked()</signal>
   <receiver>xfQuestionDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
